<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <!-- Left Panel -->
      <widget class="QWidget" name="leftPanel">
       <layout class="QVBoxLayout" name="leftLayout">
        <item>
         <!-- Search Bar -->
         <widget class="QLineEdit" name="searchBar">
          <property name="placeholderText">
           <string>Search...</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="documentsHeaderLayout">
          <item>
           <widget class="QLabel" name="documentsLabel">
            <property name="text">
             <string>Documents</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="uploadButton">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="documentList"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="chatsHeaderLayout">
          <item>
           <widget class="QLabel" name="chatsLabel">
            <property name="text">
             <string>Chats</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="newChatButton">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="chatHistoryList"/>
        </item>
       </layout>
      </widget>
      <!-- Right Panel -->
      <widget class="QWidget" name="rightPanelContainer">
       <layout class="QVBoxLayout" name="rightPanelContainerLayout">
        <item>
         <!-- Settings Button -->
         <layout class="QHBoxLayout" name="settingsButtonLayout">
          <item>
           <spacer name="spacerLeft">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint">
             <size>
              <width>1</width>
              <height>1</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="settingsButton">
            <property name="text">
             <string>âš™</string>
            </property>
            <property name="toolTip">
             <string>Settings</string>
            </property>
            <property name="maximumWidth">
             <number>40</number>
            </property>
            <property name="maximumHeight">
             <number>40</number>
            </property>
            <property name="styleSheet">
             <string>border: none; font-size: 16px;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="rightPanel">
          <layout class="QVBoxLayout" name="rightLayout">
           <item>
            <widget class="QScrollArea" name="chatScrollArea">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="chatContainer">
              <layout class="QVBoxLayout" name="chatLayout">
               <property name="alignment">
                <set>Qt::AlignTop</set>
               </property>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="inputLayout">
             <item>
              <widget class="QLineEdit" name="promptInput">
               <property name="placeholderText">
                <string>Enter your query...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="sendButton">
               <property name="text">
                <string>Send</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>